lua/wfp_verify.lua

```diff
@@ -216,14 +218,34 @@ function verifyWFP(tTarget, tWfpControl, iChiptype, atWfpConditions, tPlugin, tF
         end
     end
     -- print final verify result and return
+    print_verify_summary(atFlashDataTable, tLog)
     print("VerifyWfp result: " .. tostring(fVerified))
```

Comment:
Is this intentionally a print, or should it be a log command? Or was it temporary and should be removed (same question for several other print statements in this file).